### PR TITLE
Cleanup storybook, add homepage splash

### PR DESCRIPTION
**Pull request by archae0pteryx**
- Followed redwoods' own docs on setting up storybook
- Removed unused page storybooks
- Add main retro logo (from figma)
- Added fancy paper background (from figma)

I know these figma additions are not confirmed 100% so they can be placeholders. I wanted it to at least *feel* like we're working on something else. lol.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
   },
   "packageManager": "yarn@3.2.1",
   "scripts": {
+    "storybook": "yarn rw storybook --no-manager-cache",
```
Comment (craineum) [inline package.json:31]: @archae0pteryx Should probably add this to our template.

Comment (craineum) [reply package.json:31]: Along with the associated config.

File: web/src/components/Footer/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export { Footer } from './Footer'
```
Comment (craineum) [inline web/src/components/Footer/index.ts:1]: ❓ I thought these files weren't needed under Redwood?!?

File: web/src/components/Verification/Verification.test.tsx [hunk 3]
```diff
@@ -4,6 +4,7 @@ import { Verification } from './Verification'
 
 describe('Verification', () => {
   it('renders successfully', () => {
+    mockGraphQLMutation('VerifyEmailMutation', (_variables) => ({}))
```
Comment (craineum) [inline web/src/components/Verification/Verification.test.tsx:7]: ❓ I thought this was getting auto-loaded so we didn't have to call it here?!?

**Review by craineum — APPROVED** [review]
Couple questions, but looks good.

File: web/src/components/Verification/Verification.test.tsx (hunk 3 above)
Comment (archae0pteryx) [reply web/src/components/Verification/Verification.test.tsx:7]: The tests were shouting about it. Maybe we're missing something there.

File: web/src/components/Footer/index.ts (hunk 2 above)
Comment (craineum) [reply web/src/components/Footer/index.ts:1]: Yeah, I thought Redwood was doing some magic so you didn't have to do `components/Something/Something` without having the index. If an index file is required to have it be `components/Something` then I am okay with these.

File: web/src/components/Verification/Verification.test.tsx (hunk 3 above)
Comment (craineum) [reply web/src/components/Verification/Verification.test.tsx:7]: Hmm... yeah, between the the `jest.config.js` and `jest.setup.js` files loading the mocks, this shouldn't be needed. Maybe something is failing about that now?

**Comment by archae0pteryx** [comment]
@craineum These changes are in the new login PR. Closing this one.

**Pull request closed without merging**